package.json

```diff
@@ -3,14 +3,11 @@
 	"version": "0.2.8",
 	"license": "MIT",
 	"scripts": {
-		"start": "ng serve",
+		"start": "node scripts/generate-config.js && ng serve",
```

Comment:
Can you add script just for `generate-config`( for dev) ? 
```json
"scripts: {
   ...
   "generate-config": "node scripts/generate-config.js"
}
```

Reply:
Fixed by https://github.com/AnSyn/ansyn/pull/1304/commits/94c83ca296e283a419ae234eae2011a3f7bc6fc1